code/index.html

```diff
@@ -4,18 +4,37 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project Name</title>
-    <!-- dont forget to add a css file and link it here! -->
-    <!-- If you're using google fonts, that should be added here -->
+    <title>Penguins</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <link rel='icon' href='/favicon.ico' type='image/x-icon'>
   </head>
   <body>
-    <!-- 
-      Write your code here inside the body to make it appear in the browser. 
-      Below is an example on how to use an image tag. Copy it and add it in the body, but outside the comment tag if you want to see how it looks like on the website. Then you can change the size and other styling for it in the css file.
 
-      <img src="example-recipe-background.png" alt="Banana Bread">
 
-    -->
-    <h1>Hello there! 👋🏼</h1>
+  <div>
+    <img src="images/penguins-pattern.jpg" alt="Home" 
+        style="width: 1000px;;
+        background-position="center;
+        background-repeat="no-repeat";
+        background-size="cover";>
+  </div>      
```

Comment:
Looks very nice! 

I would suggest to move the styling of the image to CSS (line 16 to 19), using something like `class="header-image"` and then style it in CSS. I think it is good that use an alt-tag which is important for accessibility.